libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SalesforceHybridSDKManager.m

```diff
@@ -41,6 +41,10 @@ - (NSString*) pathForUserSyncsConfig;
 @implementation SalesforceHybridSDKManager
 @synthesize appConfig = _appConfig;
 
++ (void)initializeSDK {
```

Comment:
Should we just call this `init`? The `SDK` part is redundant.

Reply:
In objective-c, init is the instance level initializer (constructor) and initialize the class level initializer. So I don't think we should call it init. Having the name based on initialize makes sense.

Reply:
Ah yes. I completely forgot. That makes sense.